packages/kbn-es-archiver/src/lib/streams.ts

```diff
@@ -17,4 +17,15 @@
  * under the License.
  */
 
-export * from '../../../../src/legacy/utils/streams';
+export {
+  createFilterStream,
+  concatStreamProviders,
+  createConcatStream,
+  createIntersperseStream,
+  createListStream,
+  createPromiseFromStreams,
+  createReduceStream,
+  createSplitStream,
+  createMapStream,
+  createReplaceStream,
+} from '../../../../src/legacy/utils';
```

Comment:
Packages cannot have deps on `core`, it breaks incapsulation and TS build. @spalger what is the best approach to remove the dependency? We might move `streams` out of the `core`. Maybe in an existing or a dedicated package. I thought `kbn-dev-utils` is a good candidate, but turned out it's not: `For running integration tests in Kibana`.

Reply:
I have every intention of removing the reliance on node streams in the es archiver, but not today. For now I'll just decouple the package from the src directory by copying the stream source into the package.

Reply:
So for now I keep it as is. Do you agree?

Reply:
https://github.com/elastic/kibana/pull/76519